osm2gtfs/creators/managua/managua.json

```diff
@@ -0,0 +1,38 @@
+{
+    "query": {
+        "bbox": {
+            "n": "12.21",
+            "s": "12.04",
+            "e": "-86.12",
+            "w": "-86.38"
+        },
+        "tags": {
+            "route":"bus",
+            "network": "IRTRAMMA",
+            "public_transport:version": "2"
+        }
+    },
+    "stops": {
+        "name_without": "Parada sin nombre - Agregaselo en rutas.mapanica.net",
```

Comment:
Isn't it very long for a stop name?

Reply:
This was inspired by the no-stop-name from Accra. Does it look bad on Transportr, etc?

Reply:
I feel that it look a bit odd. What do you think @grote ?